apps/docs/src/components/code/ExpandableLayout.tsx

```diff
@@ -41,18 +49,25 @@ export const ExpandableLayout = ({ scope, code }: ExpandableLayoutProps) => {
       />
 
       {/* Preview Section */}
-      <div
-        className={clsx(
-          "p-md pt-lg bg-bgPage border rounded-inherit",
-          isExpanded && "rounded-b-0",
-        )}
+      <HvProvider
+        themes={[pentahoPlus, ds5, ds3]}
+        theme={docsTheme}
+        colorMode={docsMode}
+        cssTheme="scoped"
```

Comment:
Let's extract this to a separate component (eg. `HvDocsProvider`)?
We're repeating it quite a few times (per "layout" renderer + `Playground`) - and most likely want to keep the exact same behaviour. Also, we'll probably want to add a few more renders down the road